### PR TITLE
Final cnn with 92% accuracy

### PR DESCRIPTION
**Pull request by akaashsidhu**

### CODE REVIEW

File: project-folder/CNN.ipynb [hunk 1]
```diff
@@ -2,9 +2,17 @@
  "cells": [
```
Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p>can we make sure that all of these libraries are used? zipfile, cv2 and loadtxt are not being referenced anywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p>nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p>nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p>I think you should make it clearer in your markdown comments that this is the model which achieved 93% accuracy and the other model was only for testing (?) purposes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p>can you justify these steps?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p>remove please</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p><code></code><code></p><p>#Somehow it improved using the validation train and test that was randomly split</p><p>#Early stopping was not good, reduced learning rate better</p><p></code><code></code></p><p> </p><p>I don't think your early stopping did anything :P </p><p>You set <code>epochs=30</code> and the model trained for 30 epochs. Why do you think reduced learning rate is better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (happilyeverafter95) [inline project-folder/CNN.ipynb]: <p>you don't actually need this step since the model is already saved in memory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>
